src/OSPSuite.Core/Domain/Descriptors/NotInContainerCondition.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+
+namespace OSPSuite.Core.Domain.Descriptors
+{
+   public class NotInContainerCondition : TagCondition
+   {
+      [Obsolete("For serialization")]
+      public NotInContainerCondition()
+      {
+      }
+
+      public NotInContainerCondition(string tag) : base(tag, Constants.NOT_IN_CONTAINER)
+      {
+      }
+
+      public override IDescriptorCondition CloneCondition() => new NotInContainerCondition(Tag);
+
+      public override bool IsSatisfiedBy(EntityDescriptor entityDescriptor)
+      {
+         return !entityDescriptor.ParentContainerTags.Contains(Tag);
```

Comment:
Only when accessing the ParentContainerTag would the descriptor be computed. Since InContainer and NotInContainer will hardly be used, this should ensure that there is no performance loss